test/set_sharding_data_test.lua

```diff
@@ -53,6 +53,12 @@ local sharding_key_format = {
     {name = 'sharding_key', type = 'array', is_nullable = false}
 }
 
+local sharding_func_format = {
+    {name = 'space_name', type = 'string', is_nullable = false},
+    {name = 'sharding_func', type = 'string', is_nullable = false},
+    {name = 'lua_code', type = 'boolean', is_nullable = false}
+}
+
```

Comment:
I would describe a schema for a space with sharding functions in a README and say that space format is a part of DDL API.

Reply:
Updated

Reply:
Seems resolved. @ligurio?